bonus.md

```diff
@@ -3,11 +3,17 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+Fetch is useful over just handling with hash[:symbol] because it allows you to handle truthy and falsy defaults that won't work with just using || for defaults
 
 2. What is the disadvantage of single inheritance, and what's a possible solution to that problem?
+A disadvantage of single inheritance is that it can violate encapsulation because you are now raising potential conflicts between methods in the child class and the parent class.
+
```

Comment:
The question is about single vs multiple inheritance, not about disadvantages of inheritance in general.